wiki/markdown/service-usage.md

```diff
@@ -0,0 +1,85 @@
+| /event_types/{mp}     | GET    | mp - message protocol, required                                                                                                             |                                                                                                                                                        | This endpoint is used to obtain Eiffel event types implemented in  [Eiffel REMReM Semantics](https://github.com/eiffel-community/eiffel-remrem-semantics).                             |
+| /template/{type}/{mp} | GET    | type - Eiffel event type mp - message protocol, required                                                                                    |                                                                                                                                                        | This endpoint is used to obtain Eiffel event templates implemented in  [Eiffel REMReM Semantics](https://github.com/eiffel-community/eiffel-remrem-semantics).                         |
+| /versions             | GET    |                                                                                                                                             |                                                                                                                                                        | This endpoint is used to get versions of generate service and all loaded protocols versions  in JSON format.                                                                           |
+
+
+## Examples
+
+Typical examples of usage Eiffel REMReM Generate Service endpoints are described below.
+
+You can use command line tools like [curl](https://curl.haxx.se/) or some plugin for your favorite browser. For example:
+
+*   [Postman](https://chrome.google.com/webstore/detail/postman/fhbjgbiflinjbdggehcddcbncdddomop) for Chromium-based browsers
+*   [HttpRequester](https://addons.mozilla.org/en-US/firefox/addon/httprequester/) for Firefox
+
+### Examples for `/mp` endpoint
+
+#### Given one message in file _body-single.json_:
+
+```
+curl -XPOST -H "Content-Type: application/json" --data @body-single.json http://localhost:8080/generate-service/eiffelsemantics?msgType=eiffelactivityfinished
+```
+
+Result:
+
+```
+[{"meta":{"id":"7d8ea04d-b5a6-459f-a6fe-134e55556f0b","type":"EiffelActivityFinishedEvent","version":"1.1.0","time":1521545563096,"tags":["tag1","tag2"],"source":{"domainId":"example.domain","host":"host","name":"name","serializer":{"groupId":"com.github.Ericsson","artifactId":"eiffel-remrem-semantics","version":"0.4.0"},"uri":"http://java.sun.com/j2se/1.3/"},"security":{"sdm":{"authorIdentity":"test","encryptedDigest":"sample"}}},"data":{"outcome":{"conclusion":"SUCCESSFUL"},"persistentLogs":[{"name":"firstLog","uri":"http://myHost.com/firstLog"},{"name":"otherLog","uri":"isbn:0-486-27557-4"}],"customData":[]},"links":[{"type":"ACTIVITY_EXECUTION","target":"aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee1"}]}]
```

Comment:
I noticed that these event examples are old and still use Toulouse events. They should be updated with Agen events and the serializer could use a more up-to-date verison of remrem-semantics (since we've moved organization for remrem-repositories to eiffel-community)

Reply:
repalced the examples which are redirecting to Toulouse events.